package.json

```diff
@@ -107,12 +107,12 @@
       {
         "key": "Backspace",
         "command": "extension.vim_backspace",
-        "when": "editorTextFocus && vim.active && !inDebugRepl"
+        "when": "editorTextFocus && vim.active &&   && !inDebugRepl"
```

Comment:
you've got a double `&&` typo.

Reply:
Ah thanks. I was wondering why backspace wasn't working, and thought I needed to do some extra work elsewhere to fix it.